packages/hooks-react/src/useProtectedMedia.ts

```diff
@@ -2,15 +2,22 @@ import { useQuery } from 'react-query';
 import type { PlaylistItem } from '@jwp/ott-common/types/playlist';
 import ApiService from '@jwp/ott-common/src/services/ApiService';
 import { getModule } from '@jwp/ott-common/src/modules/container';
+import { useTranslation } from 'react-i18next';
 
 import useContentProtection from './useContentProtection';
 
 export default function useProtectedMedia(item: PlaylistItem) {
   const apiService = getModule(ApiService);
-  const contentProtectionQuery = useContentProtection('media', item.mediaid, (token, drmPolicyId) => apiService.getMediaById(item.mediaid, token, drmPolicyId));
+  const contentProtectionQuery = useContentProtection('media', item.mediaid, (token, drmPolicyId) =>
+    apiService.getMediaById(item.mediaid, language, token, drmPolicyId),
+  );
```

Comment:
The `useContentProtection` hook contains the `useQuery` for the media item. The language should be passed to that hook, so it can revalidate when the `language` changes.

Although... the media item returned there is only used for media playback, so perhaps we don't need to translate it at all.